File: prak7/src/main/java/flux/prak7/models/Note.java
package flux.prak7.models;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Column;
import org.springframework.data.relational.core.mapping.Table;

@Data
@Table("note")
public class Note {
    @Id
    private Long id;
    @Column("title")
    private String title;
    @Column("description")
    private String description;
}

File: MicroservicesSolution/src/NoteService/Models/Note.cs
﻿namespace NoteService.Models
{
    public class Note
    {
        public Guid Id { get; set; }

        public string Title { get; set; }

        public string Description { get; set; }

        public Note(string title, string description)
        {
            Id = Guid.NewGuid();
            Title = title;
            Description = description;
        }
    }
}
